### DISCUSSION TITLE
Freeze metric on dashboard's health check for partitioned tables

### DISCUSSION THREAD

**Discussion by govitu** [cybertec-postgresql/pgwatch2#786] [Q&A]
I was analyzing an issue regarding table freezing, the database studied is Gitlab, the pgwatch health check dashboard indicates 2 billion for this metric, investigating how close the database is from wrapround, no evidence emerged for this, another observation was using the pgwatch's query metric.sql it indicates that tables with the highest freezing value are of partitioned type. Therefore, according to the Postgresql documentation (https://www.postgresql.org/docs/16/sql-analyze.html) it indicates that autovacuum daemon does not process partitioned tables. So my question is whether the query should consider partitioned tables, in the condition relkind in ('r','p').

**Comment by pashagolub** [comment]
Hello. Thanks a lot for your hint.

Fixed just today in #760 :-)
